src/geo/gmaps/gmaps-cartodb-layer-group-view.js

```diff
@@ -235,7 +235,7 @@ _.extend(
     _reload: function () {
       var tileURLTemplates;
       if (this.model.hasTileURLTemplates()) {
-        tileURLTemplates = this.model.getTileURLTemplates();
+        tileURLTemplates = [this.model.getTileURLTemplate()];
```

Comment:
Have we tested this? We're not providing subdomains to the Google Maps layer so it probably doesn't know how to handle `{s}` properly... We'd probably need to replace `{s}` here.